enslig-forsorger/src/main/kotlin/no/nav/familie/kontrakter/ef/iverksett/BehandlingsstatistikkDto.kt

```diff
@@ -50,6 +50,8 @@ enum class BehandlingMetode {
     MANUELL,
     AUTOMATISK,
     BATCH,
+    TOTRINN,
```

Comment:
Er TOTRINN og G_OMREGNING en Metode eller Hendelse?
For meg tenker jeg disse to er hendelser heller enn metoder, men mulig jeg misforstår noe her?